src/master-playlist-controller.js

```diff
@@ -1287,8 +1287,21 @@ export class MasterPlaylistController extends videojs.EventTarget {
       return;
     }
 
-    const suggestedPresentationDelay = this.masterPlaylistLoader_.master.suggestedPresentationDelay;
-    const mainSeekable = Vhs.Playlist.seekable(media, expired, suggestedPresentationDelay);
+    let delay;
+
+    if (this.masterPlaylistLoader_.master.hasOwnProperty('suggestedPresentationDelay')) {
+      delay = this.masterPlaylistLoader_.master.suggestedPresentationDelay;
+    } else if (media.serverControl && media.serverControl['PART-HOLD-BACK'] && media.partTargetDuration) {
```

Comment:
ll-hls has it's own version of when a playlist should be started called `HOLD-BACK` and `PART-HOLD-BACK`